backend/LeetWars.Core/LeetWars.Core.BLL/Services/ChallengeService.cs

```diff
@@ -62,5 +62,28 @@ public async Task<ICollection<ChallengePreviewDto>> GetChallengesAsync(Challenge
             var filteredChallenges = await challenges.ToListAsync();
             return _mapper.Map<List<ChallengePreviewDto>>(filteredChallenges);
         }
+
+        public async Task<ChallengeFullDto> GetChallengeByIdAsync(long id)
+        {
+            var challenges = await _context.Challenges
+                .Where(challenge => challenge.Id == id)
```

Comment:
Where can be removed. Just add predicate to FirstOrDefault()

Reply:
But then i can't use Include() method

Reply:
Hmmmm
![image](https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/32576664/db6016a7-5ed6-4083-9a9f-b8723dfc7476)

Reply:
Fixed)